src/client/index.ts

```diff
@@ -338,6 +339,16 @@ class Client extends EventEmitter {
     return transactionUtils.convertStringToHex(string);
   }
 
+  /**
+   * Autofills missing fields in a transaction
+   *
+   * @param {Transaction} tx A transaction to autofill missing fields
+   * @returns {Transaction} An autofilled transaction
```

Comment:
nit: we're using a different doc format that the linter is enforcing

Reply:
This should be resolvable with a `yarn lint --fix`